src/assisted_installer_controller/assisted_installer_controller_test.go

```diff
@@ -314,9 +314,10 @@ var _ = Describe("installer HostRoleMaster role", func() {
 
 	Context("validating AddRouterCAToClusterCA", func() {
 		conf := ControllerConfig{
-			ClusterID:        "cluster-id",
-			URL:              "https://assisted-service.com:80",
-			OpenshiftVersion: "4.7",
+			ClusterID:           "cluster-id",
+			URL:                 "https://assisted-service.com:80",
+			OpenshiftVersion:    "4.7",
+			CheckClusterVersion: true,
```

Comment:
Shouldn't we also check a case where this is false?

Reply:
Yes, we do.
Done.